terraform/k8s/default-config/mojaloop-stateful-resources.json

```diff
@@ -340,7 +348,19 @@
             "mysql_data": {
                 "is_legacy": false,
                 "existing_secret": "mysql-central-ledger-secret",
+                "general_log": 1,
+                "slow_query_log": 1,
                 "root_password": "",
+                "default_authentication_plugin": "mysql_native_password",
+                "basedir": "/opt/bitnami/mysql",
+                "plugin_dir": "/opt/bitnami/mysql/lib/plugin",
+                "socket": "/opt/bitnami/mysql/tmp/mysql.sock",
+                "datadir": "/bitnami/mysql/data",
```

Comment:
do all of these params need to be exposed as params?

Reply:
@dfry, we need to be able to change the values if needed. Default values will continue to be the same, but users will be able to change them from implementations now.